.github/sync.yml

```diff
@@ -15,10 +15,6 @@ group:
         dest: .github/workflows/docker-test.yml
       - source: .github/workflows/pull_request.yml
         dest: .github/workflows/render-site.yml
-      - source: scripts/spell-check.R
```

Comment:
Repositories no longer need individual scripts with the use of ottr-reports github actions